en/02_quicktour.md

```diff
@@ -0,0 +1,276 @@
+      SQL("select id, name, birthday from members where id = {id}")
+        .bindByName('id -> id)
+        .map { rs => Member(rs.long("id"), rs.string("name"), rs.timestampOpt("birthday").map(_.toDateTime) }
+        .single.apply()
+    }
+
+we can write it as below. Notice that it has also become simpler as passing binding variables by `#bindByName` being no longer there.
+
+    def create(name: String, birthday: Option[LocalTime])(implicit session: DBSesion): Member = {
+      val id = sql"insert into members (name, birthday) values (${name}, ${birthday})"
+        .updateAndReturnGeneratedKey.apply()
+      Member(id, name, birthday)
+    }
+    
+    def find(id: Long)(implicit session: DBSesion): Option[Member] = {
+      sql"select id, name, birthday from members where id = ${id}"
+        .map { rs => 
+          new Member(
+            id       = rs.long("id"), 
+            name     = rs.string("name"), 
+            birthday = rs.timestampOpt("birthday").map(_.toDateTime) 
+          )
+        }
+        .single.apply()
+    }
+
+
+In the current ScalikeJDBC, the latter is the more recommended style over the direct use of `SQL("...")`. Subsequent chapters in this book basically shows code examples using SQL interpolation.
+
+### QueryDSL
+
+QueryDSL is a feature that was added in 1.6.0 which also should not be forgotten. This is a type-safe SQL builder. It will create an object of the above SQL interpolation.
+
+    import scalikejdbc._
+    
+    case class Member(id: Long, name: String, birthday: Option[LocalTime] = None)
+    object Member extends SQLSyntaxSupport[Member] {
+      override tableName = "members"
+      override columnNames = Seq("id", "name", "birthday")
+      
+      def create(name: String, birthday: Option[LocalTime])(implicit session: DBSesion): Member = {
+        val id = withSQL { 
+          insert.into(Member).namedValues(
+            column.name -> name,
+            column.birthday -> birthday
+          )
+        }.updateAndReturnGeneratedKey.apply()
+        Member(id, name, birthday)
+      }
+      
+      def find(id: Long)(implicit session: DBSesion): Option[Member] = {
+        val m = Member.syntax("m")
+        withSQL { select.from(Member as m).where.eq(m.id, id) }
+          .map { rs => 
+            new Member(
+              // rs.get[Long] can be used with type inference instead of writing rs.long
+              id       = rs.get(m.resultName.id), 
+              name     = rs.get(m.resultName.name),
+              birthday = rs.get(m.resultName.birthday)
+            ) 
+          }.single.apply()
+      }
+    }
+
+In a glance, it looks like we have more code, but we have gotten rid of most of the parts where SQL were executed by strings.
+
+As a result, even a complex query, for example using joins, can become more DRY. If you develop a certain scale of applications, QueryDSL will enhance development efficiency.
```

Comment:
If you go with this style, complex join queries can be more DRY. When you develop applications of a certain size, using QueryDSL would make your development productive.